src/persistence/local-storage/index.ts

```diff
@@ -30,14 +30,14 @@ export function getStorage(key: string): unknown {
   }
 }
 
-export function setStorageKey(key: string, storage: unknown): void {
-  localStorage.setItem(key, JSON.stringify(storage));
+export function setStorageKey(key: string, value: string): void {
+  localStorage.setItem(key, value);
```

Comment:
Why not just using a union type and keep the `JSON.stringify` centralized in one place?

```
export function setStorageKey(key: string, value: Record<string, unknown> | string): void {
  if (typeof value === "string") {
    localStorage.setItem(key, value);
  } else {
    localStorage.setItem(key, JSON.stringify(value));
  }
}
```

Reply:
I'm not sure about this, mainly because `Record<string, unknown> | string` is only a subset of the shapes that `value` could adopt. What if we want to store a `number` or a `boolean`? Then, we still would need to convert them to string in the origin. In that case, why is a Record special?

We could make this union wider, and manage each case, but we would eventually be covering `unknown`.

I prefer the current implementation, but if you 2 still prefer centralizing the `JSON.stringify` here, let me know and I do the change.

Reply:
> JSON.stringify parses booleans and numbers just fine

Yeah, this is indeed true. Ok, agreed and done!